BadHabbit/BadHabbit/Kit/NetworkKit/Request.swift

```diff
@@ -5,8 +5,38 @@
 //  Created by Nikita Gayko on 23.05.23.
 //
 
+import Foundation
+
 struct Request {
     let path: String
     let params: [AnyHashable: Any]
+    
+    private func urlQueryItems() -> [URLQueryItem] {
+        var queryItems: [URLQueryItem] = []
+        
+        params.forEach { key, value in
+            if let key = key as? String,
+               let value = value as? String {
+                queryItems.append(URLQueryItem(name: key, value: value))
+            }
+        }
+        
+        return queryItems
+    }
+    
+    func urlComponents() -> URLComponents {
+        var urlComponents = URLComponents()
+        urlComponents.path = path
+        
+        let queryItems = urlQueryItems()
+        
+        guard !queryItems.isEmpty else {
```

Comment:
Logic too complicated. Could be more simple. If we replace with `it` statement and keep only one return from function

Reply:
Resolved